src/scss/tools/mixins/_ratings.scss

```diff
@@ -0,0 +1,37 @@
+//
+// ==========================================================================
+// Ratings variant mixin
+// Used in alert modifier classes to colorize the alert
+// ==========================================================================
+//
+
+$rating-iterations: (0, 2, 5, 6, 8);
+
+@mixin rating-fill($maxStars: 6) {
+    @if $maxStars > 0 {
+        @for $i from 0 to $maxStars + 1 {
+            @if $i < $maxStars {
+                @each $ratingIteration in $rating-iterations {
+                    @if $i == 0 and $ratingIteration == 0 {
+                        &.c-ratings--fill-00 {
```

Comment:
Stylelint is failing the Travis build as the nesting on this line and line 21 exceeds the maximum indentation level. I think it's OK to wrap this mixin with `/* stylelint-disable max-nesting-depth */` and `/* stylelint-enable max-nesting-depth */`?